next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -43,28 +43,24 @@ export function MultiCheckboxFacet({
           const checked = option.selected;
           const value = option.value as FieldValue;
           return (
-            <li key={`${getFilterValueDisplay(option.value)}`}>
-              <div className="flex items-center">
-                <Checkbox
-                  data-transaction-name={`facet - ${label}`}
-                  id={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                  checked={checked}
-                  onClick={() => (checked ? onRemove(value) : onSelect(value))}
-                />
-                <label
-                  className="ml-2 text-sm text-steelgray"
-                  htmlFor={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                >
-                  {getFilterValueDisplay(option.value)}{" "}
-                  <span className="text-steelgray">
-                    ({option.count.toLocaleString("en")})
-                  </span>
-                </label>
-              </div>
+            <li
+              key={`${getFilterValueDisplay(option.value)}`}
+              className="flex items-center"
+            >
+              <Checkbox
+                data-transaction-name={`facet - ${label}`}
```

Comment:
Is this `data-transaction-name` useful here? I have a feeling it's also copied from the initial implementation and we can remove it too